tensorflow-core/tensorflow-core-api/src/test/java/org/tensorflow/SavedModelBundleTest.java

```diff
@@ -325,9 +324,11 @@ private static Signature buildGraphWithVariables(Ops tf, Shape xShape) {
     Placeholder<TFloat32> x = tf.placeholder(TFloat32.class, Placeholder.shape(xShape));
     Variable<TFloat32> y =
         tf.withName("variable")
-            .variable(tf.random.randomUniform(tf.constant(xShape), TFloat32.class));
+            .variable(
+                tf.initScope()
```

Comment:
This change seems ugly. Is there no way we can grab the op and force it into the initializer scope?

Users are going to forget that they need to put the variables into the init scope.

Reply:
Yeah, Python lifts variable initializers automatically, we can too but I'd need #302 first.  Perhaps something like `withInitScope(Function<Ops, T>)` for now?

Reply:
Well, I'd need #302 for functions, I can do this fine for now.